src/main/resources/lang/en_US.json

```diff
@@ -169,15 +170,13 @@
     "showSwordKills": "Show Sword Kills",
     "compactTabList": "Compact Tab List",
     "highlightSpecialEnchantments": "Highlight Special Enchantments",
-    "disableSpiritSceptreMessages": "Disable Ability Messages",
-    "showSpiritSceptreDisplay":  "Ability damage display",
+    "disableSpiritSceptreMessages": "Disable Damage Chat Messages",
```

Comment:
This key should be changed so that it’s treated as a new string

Reply:
Ok